### PR TITLE
Add temporary variables page in html tests

### PR DESCRIPTION
**Pull request by gretanausedaite**
## Changes

Added a page with most of the variables listed so our designer could have an easy access deployed page to look at when adding and editing colours in Figma design library.

This page can be removed later.

## Testing

N/A
## Docs

N/A

### CODE REVIEW

**Comment by mayank99** [comment]
but this already exists: https://dev-itwinui.bentley.com/docs/variables

**Comment by gretanausedaite** [comment]
> but this already exists: [dev-itwinui.bentley.com/docs/variables](https://dev-itwinui.bentley.com/docs/variables)

Dark mode only. We need light, dark and hc hex values.

File: packages/itwinui-css/backstop/tests/variables.html [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+      import "@itwin/itwinui-icons-elements/info-circular";
+      import "@itwin/itwinui-icons-elements/status-success";
+      import "@itwin/itwinui-icons-elements/status-warning";
+      import "@itwin/itwinui-icons-elements/status-error";
+      import "@itwin/itwinui-icons-elements/status-rejected";
+    </script>
+    <style>
+      @import url("./assets/demo.css") layer(demo);
+      @import url('@itwin/itwinui-variables') layer(variables);
+      @import url("@itwin/itwinui-css/css/all.css") layer(itwinui);
```
Comment (mayank99) [inline packages/itwinui-css/backstop/tests/variables.html:31]: component css doesn't need to be included i guess. only variables.

**Review by mayank99 — APPROVED** [review]

File: packages/itwinui-css/backstop/tests/variables.html (hunk 1 above)
Comment (gretanausedaite) [reply packages/itwinui-css/backstop/tests/variables.html]: Removed not needed imports for components and icons

Comment (mayank99) [reply packages/itwinui-css/backstop/tests/variables.html]: this broke some styles! i think `global.css` is needed

Comment (gretanausedaite) [reply packages/itwinui-css/backstop/tests/variables.html:24]: Should be fixed. all.css was needed

**Pull request merged**